packages/perseus-editor/src/components/interactive-graph-settings.tsx

```diff
@@ -1,7 +1,4 @@
 /* eslint-disable react/forbid-prop-types, react/no-unsafe */
-/**
- * Used in the editor for the InteractiveGraph widget.
- */
```

Comment:
I thought this comment was quite useful.

Reply:
Oops! I think I accidentally removed it in a copy/paste. I'll add it back.